### DISCUSSION TITLE
Feat: Add hardware versions to Zooz device labels

### DISCUSSION THREAD

**Discussion by zackbcom** [zwave-js/zwave-js#6886] [Feature Requests]
## Use case
It would be helpful to have the hardware version of the device available for use in the UI.
These seem to be determined by `firmwareVersion` installed or `productId`
Example hardware `v1` for `ZEN15` uses firmware starting with `1.0`, hardware `v2` use firmware `2.0`. [source](https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/#ZEN15%20Power%20Switch%201.04)

## How it might be able to be implemented.
- Config updates to change the `label` / `Product Code` to include version if it can be based on conditional version numbers.
  - ex: `ZEN15` -> `ZEN15-V1` and `ZEN15` -> `ZEN15-V2` 
  - We have seen this done for some devices for the `800LR` new versions. [source](https://github.com/zwave-js/node-zwave-js/blob/b2b647f3eea9d6045ea14b05f108a58846e54197/packages/config/config/devices/0x027a/zen04.json#L13)
  - `"$if": "firmwareVersion >= 1.6",` something like what is done for params. [[Zooz zen15 Example](https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x027a/zen15.json#L51C4-L51C36)]
- Move firmware versions into separate files based on hardware versions based on doc([Device configuration files](https://zwave-js.github.io/node-zwave-js/#/config-files/overview?id=device-configuration-files))
  -   - ex: `zen15.json` -> `zen15_1.01-1.06.json` and `zen15.json` -> `zen15_2.00-2.10.json` 
  - > `packages/config/config/devices/<manufacturerID-as-hex>/<device-name>[_<firmware-range>].json)`
 - Add another field for hardware versions.
 - Maybe something else.

**Comment by AlCalzone** [comment]
Do you have the "same" devices with different hardware versions? Z-Wave also knows a `hardwareVersion`, which we could consider using for device identification.
I'm curious if this was actually updated by Zooz between hardware revisions. You should find it in Z-Wave JS UI when you open the `Version` dropdown.

**Comment by danielsza** [comment]
![IMG_1154](https://github.com/zwave-js/node-zwave-js/assets/6549750/5e7b5f95-3f01-409d-874f-23ee45fe3dd6)
![IMG_1153](https://github.com/zwave-js/node-zwave-js/assets/6549750/9f450331-f8a0-477e-82d4-046e111c9148)

These are the only 2 zooz devices that I have that have different hardware revisions 

edit: forgot the model #
zse40  v2 (with cr123a)
Zse40 700 (v3)

**Reply by AlCalzone** [reply]
Thanks - so no way to use that as identification
